### ISSUE TITLE
Migrate metadata, redirects, aliases

### ISSUE THREAD

**Issue by stephen-cox** [localgovdrupal/localgov_legacy_migration#16]

**Comment by stephen-cox** [comment]
@andybroomfield @Adnan-cds We are not currently using the Redirect or Metatags module in LocalGov Drupal. Would you like migrations for this data? If so, I'll add the modules to localgov_core and then add migrations for them.

**Comment by Adnan-cds** [comment]
> Would you like migrations for this data? If so, I'll add the modules to localgov_core and then add migrations for them.

Yes, please.  Can't think of a Drupal site without Redirect and Metatag.  In fact you raised a [ticket](https://github.com/localgovdrupal/localgov/issues/25#issuecomment-637531432) about these early on :)

**Comment by andybroomfield** [comment]
Hi @stephen-cox, as @Adnan-cds we are quite big users of redirect an meta tag so those will need to come across.

There's actually two redirect modules, the is redirect and a ported drupal 7 module cake url router, though ideally the redirects from that should merge into the redirects table.

**Comment by Adnan-cds** [comment]
Hi Andy,
As far as I remember, this urlrouter module was needed in the early days of Redirect 8.x when it couldn't redirect to external sites.  Now that Redirect can redirect to external sites, would you mind reviewing the need for urlrouter.

FAO @andybroomfield

**Comment by andybroomfield** [comment]
@Adnan-cds oh I meant I would remove url router and migrate its redirects into the redirect module. They'd going to be a bunch in there that need to come across into redirect.

**Comment by Adnan-cds** [comment]
> I meant I would remove url router and migrate its redirects into the redirect module.

Excellent :)

**Comment by stephen-cox** [comment]
Pull request for URL paths is here: https://github.com/localgovdrupal/localgov_legacy_migration/pull/22

Redirects and and metadata are still do to.

**Comment by Adnan-cds** [comment]
Hi Andy,
Could you please confirm if the BHCC site is using any **field** of type **metatag** or not.  I cannot find any in the Croydon site.  I have checked the field list report from */admin/reports/fields* and also programmatically:

```drush php:eval "print_r(array_keys(Drupal::service('entity_field.manager')->getFieldMapByFieldType('metatag')['node']));"```

The Croydon site has quite a few Metatag config files but I cannot find any *metatag* field that needs migrating.

FAO @andybroomfield

**Comment by Adnan-cds** [comment]
Forgot to mention that the **metatag** field is used to assign [metatags per entity](https://www.drupal.org/docs/8/modules/metatag/standard-usage-scenario) whereas the Metatag configs cover the per-bundle scenario.
